src/java/picard/sam/markduplicates/MarkDuplicates.java

```diff
@@ -56,7 +56,18 @@
  */
 @CommandLineProgramProperties(
         usage = "Examines aligned records in the supplied SAM or BAM file to locate duplicate molecules. " +
-                "All records are then written to the output file with the duplicate records flagged.",
+                "All records are then written to the output file with the duplicate records flagged. <br />" +
```

Comment:
Explain what "duplicate records flagged" means. How is this flagging encoded into the file?